cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.pb.go

```diff
@@ -27,8 +27,8 @@ import (
 	_ "google.golang.org/protobuf/types/descriptorpb"
 	anypb "google.golang.org/protobuf/types/known/anypb"
 	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
-	v11 "k8s.io/api/core/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
hrmmm ... the k8s REST API imports should have dropped completely out of this file